charts/osm/values.yaml

```diff
@@ -18,6 +18,7 @@ OpenServiceMesh:
   useHTTPSIngress: false
   envoyLogLevel: error
   namespace: arc-osm-system
+  namespaces: "kube-system azure-arc arc-osm-system"
```

Comment:
I feel have a value namespace and namespaces is little confusing and overloads the namespace term. If this is used to ignore namespaces, could we rename it to `ignoreNamespaces` ?

Reply:
+1